src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -561,6 +559,31 @@ public override void Initialize(RachisConsensus parent, TransactionOperationCont
         {
             base.Initialize(parent, context);
             ItemsSchema.Create(context.Transaction.InnerTransaction, Items, 32);
+            context.Transaction.InnerTransaction.CreateTree(LocalNodeStateTreeName);
+        }
+
+        public unsafe void PutLocalState(TransactionOperationContext context, string key, BlittableJsonReaderObject value)
+        {
```

Comment:
Part of the work we need to do when we add a node (including ourselves as the seed) to the cluster, is to register our certification in the cluster.